index.rst

```diff
@@ -740,7 +740,9 @@ All of these API calls require ``admin:jupyterlab`` scope.
 
     Each of the top-level object keys is the string literal exactly as shown.
     The alias information may not be complete.
-    (In other words, the image may have additional aliases that are not shown.)
+    (In other words, the image may have additional aliases that are not
+    shown.)
+    Digest may be ``null`` or the empty string, if the image digest is not known (e.g. for an image that is only present on a remote Docker registry).
```

Comment:
It should never be the empty string. (Just in general, we should never allow empty strings in a wire protocol unless we really mean the string of length zero as a value, not a placeholder for not knowing something.)

Reply:
Fixed.